lms/djangoapps/courseware/views.py

```diff
@@ -609,14 +613,32 @@ def jump_to(request, course_id, location):
     # choose the appropriate view (and provide the necessary args) based on the
     # args provided by the redirect.
     # Rely on index to do all error handling and access control.
+
+    # this block is taken from upstream url_helpers.py - and it does not guarantee to work if anything
+    # (course_key, chapter, section, position, final_target_id) is not specified. It should be enough for prototyping
```

Comment:
Not sure I understand this comment - we are not at the prototype stage any more. That's maybe from Eugeny's original code? If so the comment should be removed and the limitations addressed.